surround/cli.py

```diff
@@ -166,7 +168,8 @@ def parse_run_args(args):
             "files" : [
                 ("{project_name}/config.yaml", "output:\n  text: Hello World"),
                 ("dodo.py", "")
-            ]
+            ],
+            "templates" : []
```

Comment:
Why is this needed?

Reply:
Because of this: https://github.com/dstil/surround/blob/b5e6f46825917093d3627adfa84b0eca2cc01fc3/surround/linter.py#L37

Reply:
Otherwise, it will throw key not found error!